[CI] SmokeTestIngestWithAllDepsClientYamlTestSuiteIT test {yaml=ingest/80_ingest_simulate/Test ingest simulate with template substitutions for component templates} failing

**Build Scans:**
- [elasticsearch-intake #11034 / part1](https://gradle-enterprise.elastic.co/s/i6c47qa3yuloe)
- [elasticsearch-periodic-platform-support #4325 / ubuntu-2004_platform-support-unix](https://gradle-enterprise.elastic.co/s/2xbe5jaurqhrc)
- [elasticsearch-pull-request #34770 / part-1](https://gradle-enterprise.elastic.co/s/ulvfp43e3tq7u)
- [elasticsearch-pull-request #34748 / part-1](https://gradle-enterprise.elastic.co/s/jpavurw77oxic)
- [elasticsearch-periodic-platform-support #4293 / windows-2016_checkpart1_platform-support-windows](https://gradle-enterprise.elastic.co/s/kg7ogjagy74v6)
- [elasticsearch-periodic-platform-support #4293 / rocky-9_platform-support-unix](https://gradle-enterprise.elastic.co/s/7zyuyqz4vpffy)
- [elasticsearch-periodic #4307 / release-tests](https://gradle-enterprise.elastic.co/s/5tr5tru7ixroa)
- [elasticsearch-periodic #4299 / release-tests](https://gradle-enterprise.elastic.co/s/qpr7qmr4aev5c)

**Reproduction Line:**
```
./gradlew ":qa:smoke-test-ingest-with-all-dependencies:yamlRestTest" --tests "org.elasticsearch.smoketest.SmokeTestIngestWithAllDepsClientYamlTestSuiteIT.test {yaml=ingest/80_ingest_simulate/Test ingest simulate with template substitutions for component templates}" -Dtests.seed=472E585E5C99A482 -Dtests.locale=en-CA -Dtests.timezone=Asia/Aqtau -Druntime.java=22
```

**Applicable branches:**
8.x

**Reproduces locally?:**
N/A

**Failure History:**
[See dashboard](https://es-delivery-stats.elastic.dev/app/dashboards#/view/dcec9e60-72ac-11ee-8f39-55975ded9e63?_g=(refreshInterval:(pause:!t,value:60000),time:(from:now-7d%2Fd,to:now))&_a=(controlGroupInput:(chainingSystem:HIERARCHICAL,controlStyle:twoLine,ignoreParentSettings:(ignoreFilters:!f,ignoreQuery:!f,ignoreTimerange:!f,ignoreValidations:!t),panels:('0c0c9cb8-ccd2-45c6-9b13-96bac4abc542':(explicitInput:(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,enhancements:(),fieldName:task.keyword,grow:!t,id:'0c0c9cb8-ccd2-45c6-9b13-96bac4abc542',searchTechnique:wildcard,selectedOptions:!(),singleSelect:!t,title:'Gradle%20Task',width:medium),grow:!t,order:0,type:optionsListControl,width:small),'144933da-5c1b-4257-a969-7f43455a7901':(explicitInput:(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,enhancements:(),fieldName:name.keyword,grow:!t,id:'144933da-5c1b-4257-a969-7f43455a7901',searchTechnique:wildcard,selectedOptions:!('test%20%7Byaml%3Dingest%2F80_ingest_simulate%2FTest%20ingest%20simulate%20with%20template%20substitutions%20for%20component%20templates%7D'),title:Test,width:medium),grow:!t,order:2,type:optionsListControl,width:medium),'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850':(explicitInput:(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,enhancements:(),fieldName:className.keyword,grow:!t,id:'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850',searchTechnique:wildcard,selectedOptions:!('org.elasticsearch.smoketest.SmokeTestIngestWithAllDepsClientYamlTestSuiteIT'),title:Suite,width:medium),grow:!t,order:1,type:optionsListControl,width:medium)))))

**Failure Message:**
```
java.lang.AssertionError: Failure at [ingest/80_ingest_simulate:376]: got unexpected warning header [
	299 Elasticsearch-8.16.0-0f0efb79e327b3ce626ee8cd88e8bf3a4730154e "index template [test-composable-1] has index patterns [foo*] matching patterns from existing older templates [global] with patterns (global => [*]); this template [test-composable-1] will take precedence during new index creation"
]

```

**Issue Reasons:**
- [8.x] 8 failures in test test {yaml=ingest/80_ingest_simulate/Test ingest simulate with template substitutions for component templates} (1.4% fail rate in 580 executions)
- [8.x] 2 failures in step part-1 (1.8% fail rate in 114 executions)
- [8.x] 2 failures in step release-tests (12.5% fail rate in 16 executions)
- [8.x] 2 failures in pipeline elasticsearch-pull-request (1.8% fail rate in 113 executions)
- [8.x] 2 failures in pipeline elasticsearch-periodic (10.5% fail rate in 19 executions)
- [8.x] 2 failures in pipeline elasticsearch-periodic-platform-support (10.5% fail rate in 19 executions)

**Note:**
This issue was created using new test triage automation. Please report issues or feedback to es-delivery.

--- a/muted-tests.yml
+++ b/muted-tests.yml
@@ -312,12 +312,6 @@ tests:
 - class: org.elasticsearch.action.search.SearchQueryThenFetchAsyncActionTests
   method: testMinimumVersionSameAsNewVersion
   issue: https://github.com/elastic/elasticsearch/issues/114593
-- class: org.elasticsearch.smoketest.SmokeTestIngestWithAllDepsClientYamlTestSuiteIT
-  method: test {yaml=ingest/80_ingest_simulate/Test ingest simulate with template substitutions for component templates}
-  issue: https://github.com/elastic/elasticsearch/issues/114602
-- class: org.elasticsearch.smoketest.SmokeTestIngestWithAllDepsClientYamlTestSuiteIT
-  method: test {yaml=ingest/80_ingest_simulate/Test ingest simulate with template substitutions for component templates removing pipelines}
-  issue: https://github.com/elastic/elasticsearch/issues/114603
 - class: org.elasticsearch.xpack.security.support.SecurityIndexManagerIntegTests
   method: testOnIndexAvailableForSearchIndexAlreadyAvailable
   issue: https://github.com/elastic/elasticsearch/issues/114608

--- a/qa/smoke-test-ingest-with-all-dependencies/src/yamlRestTest/resources/rest-api-spec/test/ingest/80_ingest_simulate.yml
+++ b/qa/smoke-test-ingest-with-all-dependencies/src/yamlRestTest/resources/rest-api-spec/test/ingest/80_ingest_simulate.yml
@@ -375,7 +375,7 @@ setup:
 
   - do:
       allowed_warnings:
-        - "index template [test-composable-1] has index patterns [tsdb_templated_*] matching patterns from existing older templates [global] with patterns (global => [*]); this template [test-composable-1] will take precedence during new index creation"
+        - "index template [test-composable-1] has index patterns [foo*] matching patterns from existing older templates [global] with patterns (global => [*]); this template [test-composable-1] will take precedence during new index creation"
       indices.put_index_template:
         name: test-composable-1
         body:
@@ -527,7 +527,7 @@ setup:
 
   - do:
       allowed_warnings:
-        - "index template [test-composable-1] has index patterns [tsdb_templated_*] matching patterns from existing older templates [global] with patterns (global => [*]); this template [test-composable-1] will take precedence during new index creation"
+        - "index template [test-composable-1] has index patterns [foo*] matching patterns from existing older templates [global] with patterns (global => [*]); this template [test-composable-1] will take precedence during new index creation"
       indices.put_index_template:
         name: test-composable-1
         body:

